statsmodels/tsa/statespace/kalman_filter.py

```diff
@@ -739,11 +738,13 @@ def set_filter_timing(self, alternate_timing=None, **kwargs):
         if 'timing_init_filtered' in kwargs:
             self.filter_timing = int(kwargs['timing_init_filtered'])
 
-    from contextlib import contextmanager
+    import contextlib
 
-    @contextmanager
+    @contextlib.contextmanager
     def fixed_scale(self, scale):
         """
+        fixed_scale(scale)
```

Comment:
What is this line communicating?

Reply:
This helps Sphinx find the right docstring and not add a contextmanager() as an attribute fo KalmanFilter and FalmanSmoother. You can always use the format to override the automatic generation sphinx woudl find, for example, if you have a func

```
def func(*args, **kwargs):
    """
    func(x, y, z=None)
    """
```

and Sphinx would pick up the first line of the docstring.